src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/LogRecordExportProcessorOptions.cs

```diff
@@ -0,0 +1,36 @@
+// <copyright file="LogRecordExportProcessorOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Logs;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs
+{
+    /// <summary>
+    /// Export processor options for logs.
+    /// </summary>
+    public class LogRecordExportProcessorOptions
+    {
+        /// <summary>
+        /// Gets or sets export processor type.
+        /// </summary>
+        public ExportProcessorType ExportProcessorType { get; set; } = ExportProcessorType.Batch;
+
+        /// <summary>
+        /// Gets or sets batch processor options.
+        /// </summary>
+        public BatchExportLogRecordProcessorOptions BatchExportProcessorOptions { get; set; } = new();
```

Comment:
This breaks `IConfiguration` integration. Here's the one I have on main-logs if you want to just take the bits: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main-logs/src/OpenTelemetry/Logs/ExportLogRecordProcessorOptions.cs